From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Fedora Python maintainers <python-devel@lists.fedoraproject.org>
Date: Wed, 15 Jul 2020 15:17:44 +0200
Subject: [PATCH] python-2.6-rpath.patch

00016 #
Remove an "-rpath $(LIBDIR)" argument from the linkage args in configure.in:
FIXME: is this for OSF, not Linux?
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index efe6922b5de..5930552a4c4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -948,7 +948,7 @@ if test $enable_shared = "yes"; then
 	  ;;
     OSF*)
 	  LDLIBRARY='libpython$(VERSION).so'
-	  BLDLIBRARY='-rpath $(LIBDIR) -L. -lpython$(VERSION)'
+	  BLDLIBRARY='-L. -lpython$(VERSION)'
 	  RUNSHARED=LD_LIBRARY_PATH=`pwd`${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
 	  ;;
     atheos*)
